src/containers/Onboarding/CreateGroup.js

```diff
@@ -40,7 +40,17 @@ class CreateGroup extends Component {
   * @param {Object} fields
   */
   _handleSubmit(fields) {
-    return this.props.createGroup(fields);
+    const promise = new Promise((resolve) => {
+      resolve(this.props.createGroup(fields));
+    });
+
+    promise.then((val) => {
+      // this.props.createOrderFrequency(val);
```

Comment:
@enricostano we saw this `createOrderFrequency` function already defined, am I right?